### ISSUE TITLE
Option to disable youtubedl_interval

### ISSUE THREAD

**Issue by labmonkey** [Jeeaaasus/youtube-dl#77]
Hello. It would be great to be able to set `youtubedl_interval=false` or similar so that the container is just run once (for example with `docker run`) instead of waiting in a loop.

**Comment by orenmazor** [comment]
I basically do this by using this docker image, but running it with my own command. this bypasses the [sleep interval](https://github.com/Jeeaaasus/youtube-dl/blob/master/root/app/youtube-dl/youtube-dl.sh#L52)

**Comment by Jeeaaasus** [comment]
Running your own command will override /init and the whole s6 process and everything no? 

I would want to implement this by telling s6-overlay to shutdown the container instead of restarting the main script, But I don't know enough to know how to accomplish that..

**Comment by orenmazor** [comment]
yup. thats the idea. I dont need any of that stuff. I just want the pre-baked image with all the converters in place, but I want to run this as a cronjob in my cluster, not as a long-lived service
